_sources/Unit1-Getting-Started/topic-1-2-java-intro.rst

```diff
@@ -120,22 +120,22 @@ You can copy the Java source code shown in this book into a file and save it if
 
 
 
-Print Commands
+Print Methods
 -------------------
 
 .. index::
    single: String
    single: String literal
 
-Java has two different print commands to print output to the screen:
+Java has two different methods to print output to the screen:
 
 - **System.out.println(value)** : prints the value followed by a new line (ln)
 - **System.out.print(value)** : prints the value without advancing to the next line
 
 
 ``System.out.println("Hi there!");`` prints out the characters between the first ``"`` and the second ``"`` followed by a new line.  The ``"Hi there!"`` is called a **string literal**, and it can have zero to many characters enclosed in starting and ending double quotes.
 
-.. activecode:: printCommands
+.. activecode:: printMethods
```

Comment:
Does changing this break anything?

Reply:
Yes, unfortunately it is not a good idea to change the ids for exercises. If they've been used in an assignment, it will cause problems on the server. There are a bunch I would love to change, because they were inherited from the old book with an old chapter number or start in numbers (which is now an error but still work) or just not named well, but it's not easy.